react-app/src/components/projects/AddProject.tsx

```diff
@@ -40,32 +41,40 @@ const AddProject = () => {
   const theme = useTheme();
   const api = usePimsApi();
   const { data: lookupData } = useContext(LookupContext);
-
+  const [tasksForAddState, setTasksForAddState] = useState([]);
   const { submit, submitting } = useDataSubmitter(api.projects.postProject);
-  const tasksForAddState = useMemo(() => {
+  const exemptionTask = useMemo(
+    () => lookupData?.Tasks.find((a) => a.Name === 'Exemption requested'),
+    [lookupData],
+  );
+  const exemptionNote = useMemo(
+    () => lookupData?.NoteTypes.find((a) => a.Name === 'Exemption'),
+    [lookupData],
+  );
+  const salesCostType = useMemo(
+    () => lookupData?.MonetaryTypes?.find((a) => a.Name === 'SalesCost'),
+    [lookupData],
+  );
+  const programCostType = useMemo(
+    () => lookupData?.MonetaryTypes?.find((a) => a.Name === 'ProgramCost'),
+    [lookupData],
+  );
```

Comment:
Similar comment as in other PR regarding finding items using name. Changing name could cause issues.

Reply:
I think I'll just handle this pattern in all the spots its used in another ticket. Will make an enum for it as I believe the id's should remain consistent enough between migrations as long as we don't screw up the sequence somehow.

Reply:
Replaced these with the relevant enums.